### ISSUE TITLE
Run Safari web tests on real Device - is it possible?

### ISSUE THREAD

**Issue by toa777** [ios-driver/ios-driver#276]
Hello Is it possible to run Safari web test on real device? Thanks

**Comment by sramgn** [comment]
Yes. Was successful in running it on iOS 7.X. The device has to be enabled with developer mode = ON. Having some issues with iOS 8.3 though.

**Comment by uparejiya** [comment]
@sramgn  How to run Safari web test on real device, can you please share method to do that?
It will be very helpful to me.

Thanks
